crates/uv-resolver/src/lock.rs

```diff
@@ -1780,6 +1782,9 @@ impl std::fmt::Display for LockError {
                     "found distribution `{id}` with development dependency group `{group}` but no base distribution",
                 )
             }
+            LockErrorKind::InvalidWheelSource { source } => {
```

Comment:
@BurntSushi Is there are reason why we implement as of those manually instead of deriving them with thiserror?